### PR TITLE
Update doc

### PR DESCRIPTION
**Pull request by weiqingy**
## What changes were proposed in this pull request?
Update doc.

### CODE REVIEW

**Comment by weiqingy** [comment]
cc: @jerryshao

File: README.md [hunk 1]
```diff
@@ -133,8 +133,8 @@ Users can include the package as the dependency in your SBT file as well. The fo
 For running in a Kerberos enabled cluster, the user has to include HBase related jars into the classpath as the HBase token
 retrieval and renewal is done by Spark, and is independent of the connector. In other words, the user needs to initiate the
 environment in the normal way, either through kinit or by providing principal/keytab.  The following examples show how to run
-in a secure cluster with both yarn-client and yarn-cluster mode. Note that SPARK_CLASSPATH has to be set for both modes, and
-the example jar is just a placeholder for Spark.
+in a secure cluster with both yarn-client and yarn-cluster mode. Note that if your Spark includes the patch of ["SPARK-20059: Use the correct classloader for HBaseCredentialProvider"](https://github.com/apache/spark/pull/17388),
```
Comment (jerryshao) [inline README.md]: I think you could simply say that SPARK_CLASSPATH is not required to set in Spark 2.1.1+.

Also be noted SPARK_CLASSPATH is deprecated, will potentially be removed. We should not encourage user to use deprecated configurations.

Comment (weiqingy) [inline README.md]: This doc is for branch-1.6 (spark 1.6.*).

Comment (jerryshao) [inline README.md]: Then providing this link to Apache Spark is a little weird. This patch for Spark1 is only existed in hwx spark1.

Comment (weiqingy) [inline README.md]: I just thought users may want to pull that patch into their own spark.

**Comment by weiqingy** [comment]
The doc has been updated based on review comments.

**Pull request merged**